server/src/main/java/org/opentosca/toscana/core/plugin/PluginFileAccess.java

```diff
@@ -88,6 +90,33 @@ public BufferedWriter access(String relativePath) throws IOException {
         }
     }
 
+    /**
+     Returns a BufferedOutputStream which can write to given path.
+     <p>
+     This method is the ByteStream Counterpart to the <code>access</code> method.
+     <p>
+     Please do not use this method to write Character encoded files,
+     the regular <code>access</code> method is intended to handle Character based Streams.
+     <p>
+     If necessary, creates missing subdirectories.
+     <p>
+     Note: Close returned BufferedOutputStream after usage.
+
+     @param relativePath path to the target file, relative to the transformations root dir
+     @return BufferedOutputStream which writes to target file.
+     @throws FileNotFoundException if given relativePath points to a directory
+     */
+    public BufferedOutputStream accessAsInputStream(String relativePath) throws IOException {
```

Comment:
Shouldn't that method be called `accessAsOutputStream`?

Reply:
Yes it should